.github/workflows/create-release-pull-request.yaml

```diff
@@ -21,10 +21,18 @@ jobs:
 
       - name: Start Changelog entry
         run: |
-          function dt {
-            date +'%Y-%m-%d'
-          }
-          sed -i "s/<!--Start Of Releases (DO NOT DELETE THIS LINE)-->/<!--Start Of Releases (DO NOT DELETE THIS LINE)-->\n\n## [${{ github.event.inputs.newVersionNumber }}] - $(date +%F)\n\n## Highlights\n\n## What’s Changed\n\n## New Contributors/" ./CHANGELOG.md
+          echo ${{ secrets.GITHUB_TOKEN }} | gh auth login --with-token
+          RELEASE_LIST=$(gh release list)
+          echo $RELEASE_LIST
+          LATEST_DRAFT_RELEASE=$(echo $RELEASE_LIST | grep Draft | head -n1 | awk '{print $1;}')
+          echo $LATEST_DRAFT_RELEASE
+          echo "" > notes.txt
+          echo "##[${{ github.event.inputs.newVersionNumber }}] - $(date +%F)" >> notes.txt
+          echo "" >> notes.txt
+          gh release view $LATEST_DRAFT_RELEASE | sed '1,/^--$/d' >> notes.txt
+          cat notes.txt
```

Comment:
I think it's ok to leave this stuff printing logs to the GitHub Actions machine, but we can remove if you prefer.

Reply:
Yeah that's fine. As long as it doesn't contain sensitive data (which is doesn't here).